src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/QualityGuidelines/AvoidMultipleEnumerations/AvoidMultipleEnumerations.cs

```diff
@@ -205,22 +221,22 @@ private void OnOperationBlockStart(OperationBlockStartAnalysisContext operationB
                 noEnumerationMethods,
```

Comment:
Please move ` var getEnumeratorSymbols = GetGetEnumeratorMethods(wellKnownTypeProvider);` on row 218 up to compilation start